signbank/dictionary/models.py

```diff
@@ -564,7 +564,7 @@ class Meta:
                                                  max_length=255, null=True, blank=True, help_text="English translation for video example 3")
     videoexample4_translation = models.CharField(_("Example video 4 English translation"),
                                                  max_length=255, null=True, blank=True, help_text="English translation for video example 4")
-
+    fingerspelling = models.NullBooleanField(_("Fingerspelling"), default=False, help_text=_("Does the sign contain fingerspelling?"))
```

Comment:
Should this just be a `BooleanField`?

Reply:
@joshmcarthur , I have made this a `BooleanField`. However, in order for this to work properly, I had to do a little change in `update.py` :slightly_smiling_face: